dynamic-demo-plugin/webpack.config.ts

```diff
@@ -9,6 +9,7 @@ const CopyWebpackPlugin = require('copy-webpack-plugin');
 const config: webpack.Configuration = {
   mode: 'development',
   context: path.resolve(__dirname, 'src'),
+  entry: {},
```

Comment:
This tells webpack "this compilation has no regular entry points" - generation of remote container entry (`plugin-entry.js`) is handled via `ConsoleRemotePlugin`.